airlock/business_logic.py

```diff
@@ -77,6 +78,12 @@ def __post_init__(self):
         if not self.root().exists():
             raise BusinessLogicLayer.WorkspaceNotFound(self.name)
 
+    def project(self):
```

Comment:
Would this be better as a property (or renamed to `get_project`)?

Reply:
That would be my instinct, but I was following @evansd's lead on the original design for PathItems and such. `root` could also be a property

Reply:
I would probably also make `root` a property, but no very strong feelings either way